### PR TITLE
Sync time conductor with plots time range

### LINKED ISSUE #1261: [Plot] Synchronize time conductor to plot zoom/pan state.
It should be possible via a button or gesture to set the bounds of the time conductor based on a plot's current pan/zoom state.

### PR DESCRIPTION
**Pull request by shefalijoshi**
When plots time range differs from the time conductor (zoom or pan), provide an action to the user to sync the time conductor's range with the plot.

Resolves #1261

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? WIP
3. Command line build passes? Y
4. Changes have been smoke-tested? Y
5. Testing instructions included? WIP

### CODE REVIEW

**Comment by charlesh88** [comment]
I have made several changes and pushed to the branch:
- Reordered buttons and changed to use v-ifs instead of :disabled;
- Fixed Pause/Play to properly toggle a single button, rather than displaying two buttons;
- Pause/Play button now has consistent look as elsewhere;
- Pausing a plot with Pause/Play now displays the Sync button;
- Added new mixin `smallerControlButtons` which is now used by object frames in layouts and Timestrip swimlanes to shrink contained view control buttons in those contexts;

**Review by charlesh88 — CHANGES_REQUESTED** [review]
Pausing the plot via Pause/Play must display the orange warning border and icon as elsewhere.

**Comment by shefalijoshi** [comment]
> Pausing the plot via Pause/Play must display the orange warning border and icon as elsewhere.

@charlesh88 This is fixed now.

**Review by charlesh88 — APPROVED** [review]
LGTM

File: src/plugins/plot/vue/single/MctPlot.vue [hunk 1]
```diff
@@ -424,19 +459,30 @@ export default {
...
-                const isUnsynced = !value && this.openmct.time.clock();
-                const domainObject = this.openmct.legacyObject(this.domainObject);
-                if (domainObject.getCapability('status')) {
-                    domainObject.getCapability('status')
-                        .set('timeconductor-unsynced', isUnsynced);
-                }
+                this.isTimeOutOfSync = value !== true;
+
+                const isUnsynced = isLocalClock && !value;
+                this.setStatus(isUnsynced);
```
Comment (akhenry) [inline src/plugins/plot/vue/single/MctPlot.vue:469]: 👍

**Review by akhenry — APPROVED** [review]
Looks good to me!

### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**